### ISSUE TITLE
508-defect-3 [COGNITION]: Veteran ID Card page has confusing UX to request VIC.

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#63034]
### Point of contact

Liz Lantz

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

...
On staging, when I'm signed in (tested with user+1) click the "request a veteran ID card" at the bottom of the application page (url is staging.va.gov/records/get-veteran-id-cards/apply/), the button label changes to `redirecting`.  Then, a new tab opens and I'm directed to  https://vicbdc.ppd.va.gov/VIC/veteran/login, where I see a generic browser error page saying `this page can't be reached`.

I tested this on production in my own non-Veteran LOA3 account, and after clicking the button, an error message was revealed below saying my records couldn't be accessed in the system right now.

Initially I thought the accessibility issue was that a redirect link was styled like a button, but after the production experience, I think this is a button that maybe does some different things depending on who is logged into the website. I'm creating this ticket to flag for your team that some UX and accessibility improvements can be made.

### Reproduction steps

1. Log into staging with `vets.gov.user+1` and visit the URL mentioned above
2. click the button at the bottom of the page that references the veteran ID card
3. Log into production with an LOA3 account of someone who is not a Veteran and visit the URL mentioned above
4. click the button at the bottom of the page that references the veteran ID card

### Proposed solution or next steps

Someone from the team with more expertise than I should evaluate if this really is a problem -maybe the redirect thing is a weird artifact of staging linking to some other prod environment?  If the behavior of the button is conditional based on the logged-in user, then perhaps those checks need to be made when the page loads rather than when the button is clicked so semantic and expected UI elements can be displayed to provide a more clear and intuitive UX.

Additional thoughts:

1. If clicking on a thing is going to redirect someone to a new page that opens in a new tab, that thing should be an anchor element, set the expectation that it'll open in a new tab, and styled to look like a link instead of a button)
2. The page a person is redirected to should be an actual page, instead of an error (maybe there's a typo in the URL?)
3. If a person is not a veteran and they attempt to request a Veteran ID card, the error message should be more informative than just "my records can't be accessed right now".  You might consider using an info alert instead of an error.  We have some examples in profile of what we say when a person tries to access military information and they're not a veteran - lmk if those would be helpful.

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [X] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else

**Comment by jacobworrell** [comment]
I'm going to connect with Liz to get more information about this ticket and also confirm this product is in our portfolio

**Comment by HeatherWidmont** [comment]
@jacobworrell do we have an update on this one? 😄

**Comment by HeatherWidmont** [comment]
@jacobworrell following up here - do we have an update?

**Comment by HeatherWidmont** [comment]
@jacobworrell will reach out to Liz

**Comment by andaleliz** [comment]
👋 Hi! I'm here! How can I help?

**Comment by jacobworrell** [comment]
@andaleliz Hi Liz, the team and I were discussing and realizing this might lay outside the scope of the BMT team and be more of an Identity/SiteWide issue.
